.github/workflows/reset-environment.yml

```diff
@@ -0,0 +1,219 @@
+      with:
+        inlineScript: |
+          Copy-AzureWebAppSqlDatabase `
+            -ResourceGroupName ${{ inputs.resource-group-name }} `
+            -WebAppName ${{ inputs.app-name }}  `
+            -SourceSlotName ${{ inputs.source-slot-name }} `
+            -DestinationSlotName ${{ inputs.destination-slot-name }} `
+            -ConnectionStringName ${{ inputs.database-connection-string-name }} `
+            -Force
+        azPSVersion: latest
+
+    - name: Scale Database
+      # v1.1.0
+      uses: Azure/powershell@38b0aa56c413239bd9dcc898d1f62986233658bc
+      with:
+        inlineScript: |
+          Set-AzureWebAppSqlDatabaseServiceObjective `
+            -ResourceGroupName ${{ inputs.resource-group-name }} `
+            -WebAppName ${{ inputs.app-name }} `
+            -SlotName ${{ inputs.destination-slot-name }} `
+            -ConnectionStringName ${{ inputs.database-master-connection-string-name }} `
+            -ServiceObjectiveName ${{ inputs.service-objective-name }}
+        azPSVersion: latest
+
+    - name: Add destination contained User to destination database
+      # v1.1.0
+      uses: Azure/powershell@38b0aa56c413239bd9dcc898d1f62986233658bc
+      with:
+        inlineScript: |
+          Install-Module sqlserver -AllowClobber -Force
+          Add-AzureWebAppSqlDatabaseContainedUser `
+            -ResourceGroupName ${{ inputs.resource-group-name }} `
+            -WebAppName ${{ inputs.app-name }} `
+            -DatabaseSlotName ${{ inputs.destination-slot-name }} `
+            -DatabaseConnectionStringName ${{ inputs.database-master-connection-string-name }} `
+            -UserConnectionStringName ${{ inputs.database-connection-string-name }}
+        azPSVersion: latest
+
+    - name: Remove source contained User from destination database
+      # v1.1.0
+      uses: Azure/powershell@38b0aa56c413239bd9dcc898d1f62986233658bc
+      with:
+        inlineScript: |
+          Remove-AzureWebAppSqlDatabaseContainedUser `
+            -ResourceGroupName ${{ inputs.resource-group-name }} `
+            -WebAppName ${{ inputs.app-name }} `
+            -DatabaseSlotName ${{ inputs.destination-slot-name }} `
+            -UserSlotName ${{ inputs.source-slot-name }} `
+            -DatabaseConnectionStringName ${{ inputs.database-master-connection-string-name }} `
+            -UserConnectionStringName ${{ inputs.database-connection-string-name }}
+        azPSVersion: latest
+
+    - name: Start Web App Slot
+      # v1.1.0
+      uses: Azure/powershell@38b0aa56c413239bd9dcc898d1f62986233658bc
+      with:
+        inlineScript: |
+          Start-AzureWebAppSlot `
+            -ResourceGroupName ${{ inputs.resource-group-name }} `
+            -WebAppName ${{ inputs.app-name }} `
+            -SlotName ${{ inputs.destination-slot-name }}
+        azPSVersion: latest
+
+    - name: Ping Web App Slot
+      run: curl --connect-timeout 100 --retry 3 --retry-delay 15 ${{ inputs.url }}
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Prefer%20to%20use%20PowerShell%207%2B%20scripts Same for `deploy-to-azure-app-service`.

Reply:
Actually I think we have a custom script for this. I will use that.